lightning/src/ln/channel.rs

```diff
@@ -2200,10 +2208,10 @@ impl<ChanSigner: ChannelKeys> Channel<ChanSigner> {
 			match err {
 				None => {
 					if update_add_htlcs.is_empty() && update_fulfill_htlcs.is_empty() && update_fail_htlcs.is_empty() && self.holding_cell_update_fee.is_none() {
-						// This should never actually happen and indicates we got some Errs back
```

Comment:
I cant leave a comment there, but three lines up I think the TODO can be dropped now.